CHANGELOG.md

```diff
@@ -63,6 +63,7 @@
 - Update packages; set minimum Node to 14 [#685](https://github.com/node-saml/passport-saml/pull/685)
 - [**security**] Check user matches logout request before reporting logout success [#619](https://github.com/node-saml/passport-saml/pull/619)
 - Remove `node-saml` code and use an import instead [#612](https://github.com/node-saml/passport-saml/pull/612)
+- Changed `wantAssertionsSigned` default value to `true`. If your IdP does not sign assertions, you should specify it explicitly to `false` in strategy configuration
```

Comment:
Do you know what PR caused this?

Reply:
I would just mark that PR with a label of `breaking-change` and then make sure the description is accurate. The thing is, we did do a semver-major bump with that, so it really should be assumed there would be breaking changes, and it is typically up to the consumer to check transient dependencies.

I think what would really help in this case would be to clean up this README to explicitly point to the node-saml README for information. In this way, it really forces people to check the correct source for documentation and the documentation won't drift.

Reply:
I'm in complete agreement with you on your two perspectives. The question is: how to proceed? I don't have the bandwidth to do more than I am. We can't even get anyone from the community to update the existing broken docs.

Reply:
We'd love to have someone help with documentation updates. Ideally, we'd have a link to the documentation over at `node-saml` for all those properties so that there isn't any duplication. Feel free to update this PR or make others PRs to advance this goal. Thank you for being willing to help with this.

Reply:
I'm sorry but sadly I won't have the time to handle that kind of work in the coming months :/ I'll close this PR as it doesn't answer well to the problem